python/ray/data/_internal/arrow_ops/transform_pyarrow.py

```diff
@@ -45,9 +45,80 @@ def take_table(
     return table
 
 
+def _concatenate_chunked_arrays(arrs: "pyarrow.ChunkedArray") -> "pyarrow.ChunkedArray":
+    """
+    Concatenate provided chunked arrays into a single chunked array.
```

Comment:
Can we call out this is not supposed to concat tensor types?

Reply:
Ah yep that's a good check + error raise to add, I'll add that!